projects/fsastorefrontlib/src/cms-components/sync-pilot/sync-pilot-connection/sync-pilot-connection.component.html

```diff
@@ -3,9 +3,9 @@
     <h3 class="table-header-title mb-0">
       {{ 'fscommon.syncPilot.needHelp' | cxTranslate }}
     </h3>
-    <div class="notice link" (click)="establishConnection(user, componentData)">
-      <h4 class="fas fa-headset table-header-value notice"></h4>
-      <p class="mb-0 notice link text-truncate">
+    <div class="link" (click)="establishConnection(user, componentData)">
+      <h4 class="fas fa-headset table-header-value"></h4>
```

Comment:
Maybe we should have this icon as an option or even have it as an input parameter coming from impex, because maybe not all customers would want this icon to represent this feature?

Reply:
Same here I would rather leave any new idea for discussion and possible future improvement.